src/redux/store.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+import { bookSlice } from './books/booksSlice';
+import { categoriesSlice } from './categories/categoriesSlice';
```

Comment:
- [ ] @Rudy-Menekam for best practices, kindly adhere to the suggestion of the previous reviewer. Since you're exporting reducers as default export, you won't import them as named import. Import them as below without curly braces `{   }`. You can equally check the comments above and stick to good naming conventions. Instead of `booksSlice`, you can rename it `bookReducer`


```
import bookSlice from './books/booksSlice';
import categoriesSlice from './categories/categoriesSlice';
```

Reply:
Thank you for your feedback, but the first reviewer said it was optional to name it bookReducer rather than book slice
and also importing them without the braces gives me the error on the image below
![Screenshot from 2023-03-08 23-51-21](https://user-images.githubusercontent.com/30177437/223869487-766d4be0-43e7-458b-8977-f5bf21c4b902.png)